### ISSUE TITLE
npm install issue- (react scripts)

### ISSUE THREAD

**Issue by shivampatil98** [insidesherpa/JPMC-tech-task-2-PY3#149]
![333](https://user-images.githubusercontent.com/64972223/81397812-abd73480-9145-11ea-9b93-860b871f4bfa.PNG)

**Comment by abhi17uec009bansal** [comment]
I am also getting same error.

**Comment by LordUttam** [comment]
Execute `npm start` in the datafeed folder.

**Comment by vskarthikprasad** [comment]
> Execute `npm start` in the datafeed folder.

I have got the same error and as you said i ran npm start in datafedd folder but it isn't working.

![image](https://user-images.githubusercontent.com/41814503/81465113-22773f00-91e5-11ea-89d5-6e39a1d9a499.png)

**Comment by LordUttam** [comment]
Ok run `npm install` again in the datafeed folder.  You might get an error like this a few lines below:
`gyp ERR! stack import sys; print "%s.%s.%s" % sys.version_info[:3];`
Just run `npm install --ignore-scripts` . Ignore the warnings. And go for `npm start` again.
It should run properly now.

**Comment by vskarthikprasad** [comment]
ignored warnings , i have done npm start in datafeed folder again but i am still getiing the error , what to do...?


![image](https://user-images.githubusercontent.com/41814503/81467724-42642e00-91f8-11ea-92dd-03f2ef387ca0.png)

**Comment by LordUttam** [comment]
Did you go for all the steps mentioned in the reply? Did you execute `npm install --ignore-scripts` ? Also you might have to restart the console.

**Comment by LordUttam** [comment]
Also are you sure you are running cmd as administrator as OP is seemingly not?

**Comment by vskarthikprasad** [comment]
yes i am running cmd as administrator and i have restarted cmd each time it failed....?

**Comment by SiddhlodaAyyar** [comment]
Watch this video and if you still get the error comment down in the youtube section and we'll try to solve it.
https://youtu.be/wRAG9i8O_44

**Comment by sahildaryani** [comment]
> Ok run `npm install` again in the datafeed folder. You might get an error like this a few lines below:
> `gyp ERR! stack import sys; print "%s.%s.%s" % sys.version_info[:3];`
> Just run `npm install --ignore-scripts` . Ignore the warnings. And go for `npm start` again.
> It should run properly now.

yes, it is working. i did all as you said npm install and then i did npm start in the datafeed folder itself and it worked.
